specs/pvs/corla.pvs

```diff
@@ -1361,28 +1347,87 @@ END audits
+%%% audit, not its risk.
+
+%%% The ``error inflation factor'' (called $\gamma$, whose value is
+%%% always over $100\%$) must be chosen. The larger the error
+%%% inflation factor, the larger the initial sample needs to be, but
+%%% the less additional counting will be required if the audit of the
+%%% sample finds errors. The specific value chosen for CDOS is
+%%% specified in \texttt{colorado\_rlas}. A typical error inflation
+%%% factor is, e.g., $110\%$.
+  error_inflation_factor: TYPE = {g: nonneg_real | 1.0 <= g} % γ
+
+%%% Finally, an ``error tolerance'' (called %\lambda$, whose value is
+%%% always under $100\%$) must be chosen. The larger the error
+%%% tolerance is, the larger the initial sample size will have to be
+%%% to give high confidence that, even though the error rate in the
+%%% \emph{sample} is relatively large, the error rate for the
+%%% \emph{contests} as a whole will be small. A typical error
+%%% tolerance is, e.g., $50\%$.
+  error_tolerance: TYPE = {l: nonneg_real | l < 1.0 } % λ
+% @note kiniry We must use l since lambda is a reserved word
+% in PVS and using non-ASCII characters such as λ is not supported in
+% PVS.
+%%% The sample size multiplier (called $\rho$) depends upon $\alpha$,
+%%% $\gamma$, and $\lambda$ through the following formula.
+  sample_size_multiplier(a: risk_limit,
+                         g: error_inflation_factor,
+                         l: error_tolerance): real =
+    -log_lisp(a) / (1/(2*g) + l*log_lisp(1-1/(2*g)))
+%%% Recall that, however the values of $\alpha$, $\gamma$, and
+%%% $\lambda$ are set, $\rho$ is determined once and for all. It does
+%%% not depend upon the margins, the number or sizes of the
+%%% individuals contests, or on the audit data.
+
+%%% Next, we need the diluted margin, called $\mu$ in the
+%%% literature. The diluted margin is the smallest margin in votes
+%%% among the contests under audit (which we will call the
+%%% \texttt{minimum\_margin}) divided by the total number of ballots
+%%% cast across all the contests under audit.\todonealmcb{Is this
+%%% true, or is it the smallest margin over all contests under audit?}
+  minimum_margin: TYPE FROM margin
 
-%%% Next, we'll formalize what is stated in the current draft rules.
-% @trace RDPR-6-Jul-2017 Section 25.2.2(A)
-  RDPR_risk_limit: TYPE = {n : nonneg_real | n <= 5}
+% @trace struck from RDPR-6-Jul-2017 old Section 25.1.5
+
+% @description The smallest reported margin in votes between the
+% reported contest winner with the least votes in the contest, and the
+% reported contest loser with the most votes in the contest, divided
+% by the number of ballots counted in that contest.
+
+% @example For example, if the voting system tabulated 10,000 ballots
+% in an audited contest, and the reported winning candidate with the
+% fewest votes received 4,000 votes, and the reported losing candidate
+% with the most votes received 3,500 votes, the diluted margin of the
+% contest is 5% [(4,000 – 3,500) / 10,000]
```

Comment:
See above.  Those that are currently in `%` with a `@trace` will be converted into margin comments.